governance.md

```diff
@@ -59,6 +59,7 @@ The Gazebo PMC currently consists of the following individuals:
 | Jenn Nguyen          | [Intrinsic](https://www.intrinsic.ai/)         | [jennuine](https://github.com/jennuine)         | Member         | PST (UTC-8)/PDT (UTC-7) |
 | Benjamin Perseghetti | [Rudis Labs](https://github.com/rudislabs)     | [bperseghetti](https://github.com/bperseghetti) | Member         | EST (UTC-5)/EDT (UTC-4) |
 | Steve Peters         | [Intrinsic](https://www.intrinsic.ai/)         | [scpeters](https://github.com/scpeters)         | Member         | PST (UTC-8)/PDT (UTC-7) |
+| Silvio Traversaro    | [Italian Institute of Technology](https://www.iit.it)  | [traversaro](https://github.com/traversaro)  | Member    | CEST (UTC+2)/CET (UTC+1) |
```

Comment:
The alphabetization is actually by last name. Mind fixing it?

Reply:
Done!